tonic-types/src/richer_error/error_details/mod.rs

```diff
@@ -120,18 +166,23 @@ impl ErrorDetails {
     }
 
     /// Get [`RetryInfo`] details, if any
-    pub fn retry_info(&self) -> Option<RetryInfo> {
-        self.retry_info.clone()
+    pub fn retry_info(&self) -> &Option<RetryInfo> {
```

Comment:
Should these be `Option<&Foo>`? This is also a breaking change so maybe we should do this later and save it?

Reply:
I did not see that v0.6.1 was released :sweat_smile: . Sorry about that. Yes, better to do this later, I pushed a commit to revert this change. Thanks!